src/components/Layout/index.js

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+import Navbar from "../Navber";
+
+const Layout = ({ children }) => {
+  const menuname = [
+    {
+      list: "Dashboard",
+      isActive: true,
+      id: 1,
+    },
+    {
+      list: "Dashboard",
+      isActive: false,
+      id: 1,
+    },
+    {
+      list: "Dashboard",
+      isActive: false,
+      id: 1,
+    },
+    {
+      list: "Dashboard",
+      isActive: false,
+      id: 1,
+    },
+  ];
+
+  const button = {
+    buttonName: "create",
+    props: {
+      type: "button",
+    },
+  };
+
+  return (
+    <div>
```

Comment:
Use a `react fragment` here, instead of `div`.

`<div>` -> `<>`
`</div>` -> `</>`